chipy_org/apps/meetings/templates/meetings/emails/rsvp_email.html

```diff
@@ -1 +1,25 @@
-To change your rsvp, click <a href="http://{{ site.domain }}{% url 'update_rsvp_with_key' key %}">here</a>
+
+Your RSVP response, {{ rsvp.response | upper }}, for {{ rsvp.meeting }} is {{ rsvp.get_status_display | upper }}.
+
+{% if rsvp.response != "declined" %}
+<p>If you no longer plan to attend this meeting, kindly 
+    
+    {% if not rsvp.user %}
+    use the link below
+    {% else %}
+    go to <a href="http://{{ site.domain }}">chipy.org</a>
+    {% endif %}
+
+to change your response to "DECLINED". 
+This will help those who still wish to attend receive a confirmation.</p>
+{% endif %}
+
+{% if rsvp.status == "wait listed" %}
+<p>If your reservation is confirmed you will be notified by email.</p>
```

Comment:
I would move this to the top of the declined block, say line 5?

```suggestion
<p>You will be notified by email if a spot becomes available and you are taken off the wait list.</p>
```